commons-vfs2/src/test/java/org/apache/commons/vfs2/impl/test/DefaultFileSystemManagerTest.java

```diff
@@ -63,4 +68,22 @@ public void testResolveFileObjectNullAbsolute() throws FileSystemException {
     public void testResolveFileNameNull() throws FileSystemException {
         VFS.getManager().resolveName((FileName) null, "../");
     }
+
+
+    @Test
+    public void testCreateGzipFileSystem() throws IOException {
+        // setup .gz file for test
+        final File gzFile = new File("src/test/resources/test-data/test.gz");
+        final File newGzFile = File.createTempFile(getClass().getSimpleName(), ".gz");
+        newGzFile.deleteOnExit();
+        FileUtils.copyFile(gzFile, newGzFile);
+        FileSystemManager manager = VFS.getManager();
```

Comment:
May you please either add a comment as to why this test needs its own copy of the file, or simplify the test. Can't the test use the file fixture as it stands? Assuming the test releasing resources works and there are no bugs at that level of course.

Reply:
Sure. Will push again avoiding copying the file.